user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+setting of the entity fields. 
+
+
+Click the arrow on a filter to choose required values. Available value setting depend on the field type. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_filters_define.png
+
+|
+
+If more than one filter is active, only the records that meet requirements of ALL the filters are displayed. For 
+example, we have filtered the *"All Contacts"* grid so that it would only show Jameses Browns from California.
+
+.. image:: ./img/data_management/grid/grid_02.png
+
+
+Change Sorting
+^^^^^^^^^^^^^^
+
+Be default the grids are sorted by the first column in the ascending order. You can change the sorting by any field and 
+in any order - it is enough to click the column header. An upward or downward looking arrow will appear by the column 
+name.
+
+
+.. _user-guide-ui-components-grid-customized:
+
+Save Your Customized Grid View and Share It with Other Users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If there is a frequent set of filters and/or ordering that you need to use, you can save your 
+customized grid view with all the filters and return to 
+it at any moment. You can have any number of such views. This is very convenient, for example, if 
+you are working with the customers from different stores, or contacts from different states, and so on.
+
+
+You can save a set of filters and sorting applied to the grid in a named view.
+
+To save a view:
+
+- Adjust the grid
+
+- Click the :guilabel:`Options` link next to grid view name, choose :guilabel:`Save As` 
+
+|
+
+.. image:: ./img/data_management/grid/grid_custom_view_01.png
+
+|
+
+- Define the view name and click the :guilabel:`Save` button. Keep the name meaningful, so it is easy to find the grid 
+  that you need
+
+|
+
+.. image:: ./img/data_management/grid/grid_custom_view_02.png
+
+|
+
+The view will now be available in the drop-down next to the grid title
+
+|
+
+.. image:: ./img/data_management/grid/grid_custom_view_03.png
+
+|    
+
+For each of the grid views, there is a number of options
+
+.. image:: ./img/data_management/grid/grid_custom_view_04.png
+
+- You can create a new grid view based on the existing one with *"Save as"*
+- You can rename an existing grid view with *"Rename"*
+- You can share the grid view with other users with *"Share"*. This means that other users will see your customized 
+  grid view in their grid view selector.
+- You can delete the grid view
+
+.. _user-guide-ui-components-grid-action-buttons:
+
+Other Grid Controls
+^^^^^^^^^^^^^^^^^^^
+
+You can adjust any grid view with action buttons - displayed right above the grid.
+
+|
+
+.. image:: ./img/data_management/grid/grid_action_buttons.png
+
+|
+
+- Change the amount of items displayed per page: :guilabel:`View Per Page`
+
+- Scroll pages (if there is more than one page of records): |ScrollPage|
+
+- Refresh the grid, i.e. get the newest details on the displayed records: |BRefresh|
```

Comment:
Edit: Refresh the grid (i.e., get the newest details on the displayed records): |BRefresh|